ultra/tests/test_train.py

```diff
@@ -41,6 +42,38 @@ class TrainTest(unittest.TestCase):
     # Put generated files and folders in this directory.
     OUTPUT_DIRECTORY = "tests/train_test/"
 
+    @classmethod
+    def setUpClass(cls):
+        """Generate single agent model"""
+        single_agent_model_log_dir = os.path.join(
+            TrainTest.OUTPUT_DIRECTORY, "single_agent_model_logs/"
+        )
+        try:
+            os.system(
+                f"python ultra/train.py --task 00 --level easy --episodes 3 --eval-rate 2 --max-episode-steps 2 --log-dir {single_agent_model_log_dir} --eval-episodes 1 --headless"
+            )
+        except Exception as err:
+            print(err)
+            self.assertTrue(False)
+
+        if not os.path.exists(single_agent_model_log_dir):
+            self.assertTrue(False)
```

Comment:
There is no `self` attribute in this method (see also the multi-agent training command).

Reply:
Sorry, I forgot to commit those changes. It should be there now